### ISSUE TITLE
V2. Dropdown regression by the latest updates

### ISSUE THREAD

**Issue by nogever** [grommet/grommet#2166]
### Expected Behavior

Dropdown should render the items in the drop box.

### Actual Behavior

Dropdown renders the items outside/below the drop box.

### URL, screen shot, or Codepen exhibiting the issue
Related code https://github.com/grommet/grommet/blob/NEXT/src/js/components/Drop/StyledDrop.js#L50
I noticed simply removing this line fixes DropButton but causes Select to show this issue.
<img width="203" alt="screen shot 2018-08-02 at 4 07 13 pm" src="https://user-images.githubusercontent.com/3888294/43965397-1a388b72-9c8d-11e8-992e-9e7c056513fc.png">

### Steps to Reproduce
 1. 
 2. 
 3. 

### Your Environment

* Grommet version: The latest.
* Browser Name and version: IE 11
* Operating System and version (desktop or mobile):

**Comment by theRemix** [comment]
> Dropdown renders the items outside/below the drop box.

@nogever I tested the issue using the following

### Environment

- Grommet version: [NEXT](https://github.com/grommet/grommet/tree/NEXT)
- Browser Name and version: IE 11.165.17134.0 (Compatibility View disabled)
- OS: Windows 10 Home Version 1803 OS build 17134.165

### Tests

On this page http://storybook.grommet.io/?selectedKind=DropButton&selectedStory=Simple&full=0&addons=1&stories=1&panelRight=0

![storybook ie11](https://user-images.githubusercontent.com/132562/44288675-fc965380-a225-11e8-9509-e29b9759871c.png)


### Steps to reproduce

Can you provide a code sample that will reproduce the issue "Dropdown renders the items outside/below the drop box."

**Comment by srockk** [comment]
@theRemix Thanks for looking into this. I just checked now and I don't see this issue any more. Also, I noticed there's been some new patches and one of them must have fixed this issue. 
<img width="418" alt="screen shot 2018-08-17 at 5 36 28 pm" src="https://user-images.githubusercontent.com/8532807/44290052-e6e45680-a244-11e8-8675-a5398be89d57.png">
I'll close this and will open a new one if I see this again.
Thanks!
